lib/chef-dk/skeletons/code_generator/metadata.rb

```diff
@@ -5,4 +5,3 @@
 description      'Generates Chef code for Chef DK'
 long_description 'Generates Chef code for Chef DK'
 version          '0.1.0'
```

Comment:
I'm torn on that. Since we don't really upload it to a Chef Server, and the cookbook itself is already versioned with the ChefDK repo itself, I don't see a huge benefit to the overhead (unless you can think of something).